tests/native/django/test_tools/test_migrations/test_templatetags.py

```diff
@@ -64,18 +70,27 @@
     {% endwith %}
 {% endwith %}
 
-{% t counter='something'|length name=name %}
+{% t counter='something'|length %}
 {counter, plural, one {
 There is only one {name} object.
 } other {
 There are {counter} {name} objects.
 }}
 {% endt %}
 
```

Comment:
No, because it will get it from the context. We know the variable should exist in the context because the blocktrans this originated from also didn't have `name` in its params.